catroid/src/createAtSchool/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -48,6 +48,7 @@
 	public static final String LIBRARY_BACKGROUNDS_URL_PORTRAIT = BASE_URL_HTTPS + "media-library/backgrounds-portrait";
 	public static final String LIBRARY_BACKGROUNDS_URL_LANDSCAPE = BASE_URL_HTTPS + "media-library/backgrounds-landscape";
 	public static final String LIBRARY_SOUNDS_URL = BASE_URL_HTTPS + "media-library/sounds";
+	public static final String CATROBAT_PRIVACY_POLICY_URL = "https://catrob.at/privacypolicy";
```

Comment:
```suggestion
	public static final String CATROBAT_PRIVACY_POLICY_URL = "https://catrob.at/createatschoolprivacy";
```
See description in https://jira.catrob.at/browse/CATROID-804